packages/clay-color-picker/src/Custom.tsx

```diff
@@ -0,0 +1,249 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import GradientSelector from './GradientSelector';
+import Hue from './Hue';
+import React, {useState} from 'react';
+import Splotch from './Splotch';
+import tinycolor from 'tinycolor2';
+import {useHexInput} from './hooks';
+
+/**
+ * Renders custom color icon
+ * @return {React.Component}
+ */
+const CustomColorIcon: React.SFC = () => {
```

Comment:
I'm thinking if maybe we can remove the `return` in these cases from functions with just the JSX and use this as default, something like this:
```typescript
const CustomColorIcon: React.SFC = () => (
  <svg ...>
     ...
  </svg>
);
```

what do you think?